ngraph/test/runtime/interpreter/unit_test.manifest

```diff
@@ -14,6 +14,9 @@ reduce_sum_keep_large_1d_to_scalar
 INTERPRETER.onnx_resize11_scales_nearest_asymmetric_floor_dynamic_sizes
 INTERPRETER.onnx_resize11_scales_down_linear
 INTERPRETER.interpolate_down_scales_const_linear
+INTERPRETER.onnx_resize10_up_scales_const_nearest
```

Comment:
Why do you disable these tests?

Reply:
Because MyriadX plugin doesn't support Interpolate,Myriad plugin doesn't support Interpolate but support its special cases (Resample or Resize) and previously it was converted to some of these layers successfully, but after merge of PR https://github.com/openvinotoolkit/openvino/pull/1412, ONNX Importer conversion of Faster_RCNN is failed.

The reason of this failure was conversion of Resize-10 into Interpolate-4, not in Interpolate-1. As a result, I reverted changes in the conversion of Resize-10.

But, after this reversion, these test began to fail. And more, disabled tests require the method `evaluate()`, but Interpolate-1 has no method `evaluate()`. Hence, these tests cannot pass after reversion.